backend/server/services/HelloService/index.js

```diff
@@ -0,0 +1,15 @@
+const admin = require('firebase-admin');
+
+const handle = (req, res) => {
+  console.log('Recieved a hello, sending one back');
```

Comment:
I can just push the basic stuff, the directories won't be included but the biggest thing here is that we get the website in another directory so we don't have merge problems. I'll commit again and remove the mock service